frontend/playwright/ui/specs/workspace.spec.js

```diff
@@ -67,6 +67,31 @@ test("User adds a library and its automatically selected in the color palette",
   await expect(workspacePage.palette.getByText('There are no color styles in your library yet')).toBeVisible();
 });
 
+test("BUG 7760 - Layout losing properties when changing parents", async ({ page }) => {
```

Comment:
This test should belong in the `design-tab.spec` no?

Reply:
ok